priv/data.yml

```diff
@@ -1,2 +1,3 @@
-- Lorem ipsum
-- Dolor sit amet
+- "ae meus geloucos(as) :alien:, td certo? Este final de semana estava assistindo a última temporada de Bojack Horseman na Netflix. Sem spoilers claro, mas a série toca muito sobre relações tóxicas, seja com trabalho, outras pessoas, familia e nós mesmo. O Bojack, o personagem principal, tem uma relação muito problematica com a familia, onde buscava muito aprovação de sua mãe devido ao descaso em sua criação e o relacionação entre seus pais que estavam juntos por obrigação e viviam um relacionamento extremamento tóxico e abusivo. Recomendo muito essa série apesar dela ser muito pesada em sua criticas, ela tem uma veia comica muito boa. Sabem oque mais precisa de aprovação? Isso mesmo meu PR!"
+- "Boa noite meus queridos baixinhos e baixinhas da kaiser, tudo bem? Vocês sabiam que no jogo do homem aranha para ps4 que saiu este ano, um menino pediu a Insomniac, desenvolvedora do jogo, para que colocasse uma mensagem de pedido de casamento para, na época, sua namorada. A desenvolvedora ouviu o jovem e colou a mensagem em um billboard de um cinema de rua. O problema é que o relacionamento acabou alguma semanas antes do jogo ser lançado e a mensagem saiu junto com o jogo, tornando-se um dos easter eggs mais tristes da historia recente. Vale ressaltar que a garota se pronunciou em relação a humores de que ela havia deixado o garoto para ficar com o irmão dele, e apesar de ela de fato ter ficado com o irmão, não foi por isso que eles terminaram. Segue imagens ao vivo da faixada do cinema."
+- "Final de quarter não é fácil né? A gente começa a correr atrás dos cards que temos para entregar. Isso me lembra do filme `Como treinar seu dragão`, em que algumas coisas parecem difíceis de se enfrentar mas quando perdemos o medo e enfrentamos nossos medos, descobrimos que nem tudo é tão assustador assim né. É que nem o meu PR, parece difícil de revisar mas não é tanto assim ♥. Ajuda a gente, jovens ♥"
```

Comment:
Olá meus princesos e princesas :slightly_smiling_face:
Não sei se vcs já assistiram aquele filme do ano de 2000 chamado “A corrente do bem” em que um professor faz um desafio aos seus alunos em uma de suas aulas onde ele quer que eles criem algo que possa mudar o mundo. Então um de seus alunos cria um jogo chamado “pay it forward” em que cada favor que alguém recebe, ele retribui para mais três outras pessoas. Lindo isso, não? O que acha de fazermos a nossa corrente do bem também e ajudarmos nosso querido amigo que está com um pull request aberto e sem nenhum review ainda, coitado :disappointed: